R/map.R

```diff
@@ -26,7 +28,9 @@
 #'   \code{walk()} (invisibly) the input \code{.x}. It's called primarily for
 #'   its side effects, but this makes it easier to combine in a pipe.
 #' @seealso \code{\link{map2}()} and \code{\link{pmap}()} to map over multiple
-#'   inputs simulatenously
+#'   inputs simultaneously. \code{\link{as_function}()} for passing additional
+#'   arguments to \code{.f} via \code{...}, such as \code{.null} when \code{.f}
+#'   is a character or integer extractor.
```

Comment:
> Alternatively (and this would be less work), map_ should link to as_function() which can explain in more detail there.

Depending on what you mean, I already did that.